Packs/Opsgeniev2/pack_metadata.json

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "Opsgenie v2",
-    "description": "Create and read alerts, schedules and on-call information from OpsGenie.",
+    "name": "Opsgenie v2 (Deprecated)",
+    "description": "Deprecated. Use OpsGenie instead.",
```

Comment:
write v3 here

Reply:
but the name of the pack is `OpsGenie` and not `OpsGenieV3`